strawberry/ext/mypy_plugin.py

```diff
@@ -254,18 +256,19 @@ def enum_hook(ctx: DynamicClassDefContext) -> None:
 
 
 def add_method_to_class(
-    api: SemanticAnalyzerPluginInterface,
+    api: Union[SemanticAnalyzerPluginInterface, CheckerPluginInterface],
     cls: ClassDef,
     name: str,
     args: List[Argument],
     return_type: Type,
     self_type: Optional[Type] = None,
-    tvar_def: Optional[TypeVarDef] = None,
+    tvar_def: Optional[TypeVarType] = None,
```

Comment:
Admittedly, I don't know much about what's going on in this file, but what was this change here for? I know there's a bit of complexity with backwards compatibility around this symbol name here: https://github.com/strawberry-graphql/strawberry/blob/2d7a12f698f57d7bac893a939c46be011ef2536f/strawberry/ext/mypy_plugin.py#L63-L67

Is this change cognizant of that?

Reply:
hmm let me double check over the weekend on mypy <= 0.920

Reply:
Here's where this was introduced:
https://github.com/strawberry-graphql/strawberry/pull/1503

Reply:
ok tried on mypy 0.800, it still works :)